pkg/apis/kops/networking.go

```diff
@@ -37,22 +37,22 @@ type NetworkingSpec struct {
 	GCE        *GCENetworkingSpec        `json:"gce,omitempty"`
 }
 
-// ClassicNetworkingSpec is the specification of classic networking mode, integrated into kubernetes
+// ClassicNetworkingSpec is the specification of classic networking mode, integrated into kubernetes.
+// Support been removed since kops 1.4.
```

Comment:
No, it was Kubernetes 1.4. I don't think this information is relevant.

Reply:
Thanks. There are quite a few issues about when features are added/removed from kops. And given that we refer users to the spec docs, I think this is relevant to many. At least it doesn't hurt.

Reply:
Resolved